go/cmd/vtworker/vtworker.go

```diff
@@ -41,6 +42,7 @@ import (
 var (
 	cell                   = flag.String("cell", "", "cell to pick servers from")
 	commandDisplayInterval = flag.Duration("command_display_interval", time.Second, "Interval between each status update when vtworker is executing a single command from the command line")
+	immediateCaller        = flag.String("callerid", "", "immediate CallerID to set when querying vttablet")
```

Comment:
nit - I would call this flag the same way as vtctl, so it is consistent:
```
   username := subFlags.String("username", "", "If set, value is set as immediate caller id in the request and used by vttablet for TableACL check")
    if *username != "" {
        ctx = callerid.NewContext(ctx,
            callerid.NewEffectiveCallerID("vtctl", "" /* component */, "" /* subComponent */),
            callerid.NewImmediateCallerID(*username))
    }
```

Reply:
Renamed the flag to "username" and copied the flag description from vtctl